21/day01/src/main.rs

```diff
@@ -0,0 +1,41 @@
+fn main() {
+    let i = include_str!("input.txt");
+    let mut s = i.split('\n');
+    let mut x = s.next().unwrap();
+    let mut y = s.next().unwrap();
+    let mut c = 0;
```

Comment:
Combine the variables so that there is an expression before the operator and an expression after the operator, so that only two expressions are needed.

Reply:
I don't understand this either. What operator, what kind of expressions? What does combining the varriables mean?